nucliadb/nucliadb/writer/api/v1/export_import.py

```diff
@@ -17,29 +17,55 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
+from uuid import uuid4
+
 from fastapi_versioning import version
 from starlette.requests import Request
 
 from nucliadb.common.context.fastapi import get_app_context
 from nucliadb.export_import import importer
-from nucliadb.export_import.fastapi import FastAPIExportStream
+from nucliadb.export_import.utils import IteratorExportStream
 from nucliadb.writer.api.v1.router import KB_PREFIX, api
+from nucliadb_models.export_import import CreateExportResponse, CreateImportResponse
 from nucliadb_models.resource import NucliaDBRoles
 from nucliadb_utils.authentication import requires_one
 
 
+@api.post(
+    f"/{KB_PREFIX}/{{kbid}}/export",
+    status_code=200,
+    name="Start an export of a Knowledge Box",
+    tags=["Knowledge Boxes"],
+)
+@requires_one([NucliaDBRoles.MANAGER, NucliaDBRoles.WRITER])
+@version(1)
+async def start_kb_export_endpoint(request: Request, kbid: str) -> CreateExportResponse:
+    export_id = uuid4().hex
+    response = CreateExportResponse(export_id=export_id)
+    return response
```

Comment:
Is kind of confusing for me having an `/export` enpoint in both reader and writer to start and download an export. The purpose of the writer's `export` is to start an asynchronous export?

Reply:
Yes! I know... we could just have an export/import HTTP service right? but for now we decided to include in the current reader and writer.

And yes, the purpose of the writer's export is to start the job (PR coming up..)